File: FurnitureAPI/FurnitureAPI/Models/Furniture.cs
using FurnitureAPI.Data.Models;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace FurnitureAPI.Models
{
    public class Furniture
    {
   
        public int Id { get; set; }

        public string Name { get; set; }
  
        public float Price { get; set; }

        public int FurnitureTypeId { get; set; }
        public string FurnitureTypeName { get; set; }

        public string Description { get; set; }

        public string Size { get; set; }

        public bool IsDeleted { get; set; }
        public int AvailableQuantity { get; set; }
        public string ImageUrl { get; set; }

        public float RatingAverage { get; set; }
        public int NumberOfReviews { get; set; }
        public int DiscountPercentage { get; set; }
        public int OrdersCount { get; set; }

    }
}

File: FurnitureAPP/app/src/main/java/com/example/furnitureapp/models/Furniture.kt
package com.example.furnitureapp.models

import com.google.gson.annotations.SerializedName
import java.io.FileDescriptor

data class Furniture(
    val id: Int,
    val name: String,
    val imageUrl: String,
    val price: Float,
    val furnitureTypeId: Int,
    val furnitureTypeName: String,
    val description: String,
    val size: String,
    val availableQuantity: Int,
    val ratingAverage: Float,
    val numberOfReviews: Int,
    val discountPercentage: Int,
    val ordersCount: Int,
    val isDeleted: Boolean,
    val glbPath: String?,
    val glbScale: Float
)
